Core/Object Arts/Dolphin/Base/BootSessionManager.cls

```diff
@@ -292,6 +292,6 @@ updateBootImage
 imageExtension
 	"Answer the suffix for an executable image file"
 
-	^'img'! !
+	^'img7'.! !
```

Comment:
If this is required it should really be copied into the PreBoot.St script as it belongs to the BootSessionManager that performs the boot.